fastdds_monitor.pro

```diff
@@ -96,3 +96,7 @@ else:unix: LIBS += -L$$PWD/../../install/fastrtps/lib/ -lfastrtps
 
 INCLUDEPATH += $$PWD/../../install/fastrtps/include
 DEPENDPATH += $$PWD/../../install/fastrtps/include
+
+# Link to the Fast DDS Monitor includes
```

Comment:
Should not this replace lines 81 and 82?

Reply:
These are the fastdds includes. The others are the fastdds monitor ones

Reply:
These are the fastdds includes. The others are the fastdds monitor ones